NOTICE

```diff
@@ -13,7 +13,7 @@ under the licensing terms detailed in LICENSE:
 * Joshua Tenner <tenner.joshua@gmail.com>
 * Nidin Vinayakan <01@01alchemist.com>
 * Aaron Turner <aaron@aaronthedev.com>
-* Willem Wyndham <willem@cs.umd.edu>
```

Comment:
I believe this doesn't play well with the `contributors` list generated for package.json.

Reply:
Yeah this was old.  Before it was fixed this was the only way to make CI happy.